server/repositories/cmsQueries/categoryContentQuery.js

```diff
@@ -26,6 +27,14 @@ class CategoryContentQuery {
     this.query = `${queryWithoutOffset}&${getPagination(page, limit)}`;
   }
 
+  getKey() {
+    getCmsCacheKey('categoryContent', this.establishmentName);
+  }
+
+  getExpiry() {
+    return 3600;
```

Comment:
This is content on a category page?  I think caching for one hour will cause confusion (i.e. I added a piece of content, where is it?  Or I had to remove something, but it's still there).

The one to cache would be the sub_terms request.  As this just displays series and subcategories, and so won't need to be updated as regularly.  Also DCM's don't add new series/categories, only studio team do, and they can have a better understanding about the caching.

Reply:
I think we go through each call and agree what caches and for how long.